pkg/identityserver/gormstore/client.go

```diff
@@ -52,6 +52,8 @@ type Client struct {
 
 	Grants Grants `gorm:"type:INT ARRAY"`
 	Rights Rights `gorm:"type:INT ARRAY"`
+
+	TieAccessToSession bool `gorm:"not null"`
```

Comment:
If we add a `not null` field, we need to set a `default`, otherwise the migration will fail.